src/tip.js

```diff
@@ -5,6 +5,25 @@ const removeIndentation = require("remark-parse/lib/util/remove-indentation");
 const RE = /^!!! ?([\w-]+)(?: "(.*?)")?(?: <(.*?)>)?\n/;
 const TIP = "tip";
 
+const ICON_CLASS = {
+  tip: "fa fa-lightbulb-o",
+  discussion: "fa fa-comments",
+  slide: "fa fa-list-alt",
+  assessment: "fa fa-check-circle",
+  content: "fa fa-mortar-board",
+  say: "fa fa-microphone",
+  guide: "fa fa-pencil-square-o"
```

Comment:
no title for `guide`?

Reply:
[nope!](https://github.com/code-dot-org/curriculumbuilder/blob/13e492664c27cb8a2c94d49a5d96bac264869da8/curriculumBuilder/tips.py#L106)

Reply:
Nevermind, read the Python code linked in the description.